test/recipes/80-test_ssl_old.t

```diff
@@ -445,6 +443,8 @@ sub testssl {
                 $flag = "-ssl3";
             } elsif ($protocol eq "TLSv1.2") {
                 $flag = "-tls1_2";
+            } elsif ($protocol eq "TLSv1.3") {
```

Comment:
This works because TLSv1.3 is the default when TLSv1.3 is enabled, if we don't specify a protocol version? Please add a comment.